packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -528,6 +546,14 @@ class AppLayout extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElemen
     this.__updateDrawerAriaAttributes();
   }
 
+  /** @private */
+  __isDrawerAnimatingChanged(isAnimating) {
```

Comment:
Nit: We could remove the observer and move the logic into the `transitionend` listener.

Reply:
Good point, thanks. Done.